staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/compiler.go

```diff
@@ -52,17 +58,17 @@ func getEnvs() (*envs, error) {
 			initEnvsErr = err
 			return
 		}
-		noParams, err := buildNoParamsEnv(base)
+		noParams, err := buildNoParamsEnv(base, typeOverwrite{})
```

Comment:
Since func buildEnv is taking bool hasParams now, it can be directly called here?

Reply:
Yes. More on that within just-push commit.